src/averageAges.js

```diff
@@ -14,12 +14,31 @@
  *
  * @return {number}
  */
+const MALE_SEX_VALUE = 'm';
+const FEMALE_SEX_VALUE = 'f';
+
+function getAverageAge(array) {
+  const averageAge = Math.round(array.reduce(
+    (prev, { died, born }) =>
+      prev + died - born, 0) * 100 / array.length) / 100;
+
+  return averageAge;
+}
```

Comment:
Good job! I suggest considering such an option, which will be a little easier to read and maintain.
```suggestion
function getAverageAge(people) {
  const totalAge = people.reduce((acc, { died, born }) => acc + died - born, 0);
  return Math.round(totalAge * 100 / people.length) / 100;
}
```